docs/jitserver_tuning.md

```diff
@@ -42,11 +42,14 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  To enable this feature, specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command line option, both at the server and at the client JVM.
 
- A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). The default is to use a nameless cache.
+ A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache called `default`.
+
+ The maximum amount of memory that an AOT cache can consume at the server is 300 MB. Since each cache consumes memory, the number of caches that a JITServer can hold is limited by the memory available.
```

Comment:
Personally, I would remove the sentence "Since each cache consumes memory, the number of caches that a JITServer can hold is limited by the memory available." since it doesn't give any indication on the approximate number.
I would link the two paragraphs together like this:
"The maximum amount of memory that an AOT cache can consume at the server is 300 MB, but this value can be changed with the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. Once the AOT cache size limit is reached, new clients will not be able to create new AOT cache instances or add new compiled methods to the existing AOT cache instances."

Reply:
I'd also clarify that 300 MB is by default.